### PR TITLE
fix BCE loss formula visualization

### PR DESCRIPTION
**Pull request by edgarriba**
This patch fixes BCE loss formula visualization with MathJax

### CODE REVIEW

**Review by apaszke — COMMENTED** [review]
I think it's better to use `.. math::` istead of code blocks. Just add a space, someone forgot to do that.

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -182,13 +182,13 @@ class MSELoss(_Loss):
 
 class BCELoss(_WeightedLoss):
     r"""Creates a criterion that measures the Binary Cross Entropy
-    between the target and the output:
+    between the target and the output::
```
Comment (apaszke) [inline torch/nn/modules/loss.py]: Can you remove that extra colon? I'm not sure if it won't break it.

**Comment by soumith** [comment]
removed extra colon and merged

**Comment by edgarriba** [comment]
Oh thx!

**Pull request merged**